<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-02-01T11:53:50 IST" name="Suite" finished-at="2022-02-01T11:55:30 IST" duration-ms="99545">
    <groups>
      <group name="smoke">
        <method signature="IndexPageTest.verifyTitle()[pri:0, instance:com.mystore.testscripts.IndexPageTest@16eb3ea3]" name="verifyTitle" class="com.mystore.testscripts.IndexPageTest"/>
      </group> <!-- smoke -->
    </groups>
    <test started-at="2022-02-01T11:53:50 IST" name="IndexPage" finished-at="2022-02-01T11:55:30 IST" duration-ms="99545">
      <class name="com.mystore.testscripts.IndexPageTest">
        <test-method is-config="true" signature="loadConfig()[pri:0, instance:com.mystore.testscripts.IndexPageTest@16eb3ea3]" started-at="2022-02-01T11:53:50 IST" name="loadConfig" finished-at="2022-02-01T11:53:50 IST" duration-ms="458" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadConfig -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.mystore.testscripts.IndexPageTest@16eb3ea3]" started-at="2022-02-01T11:53:50 IST" name="setup" finished-at="2022-02-01T11:55:27 IST" duration-ms="96069" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyTitle()[pri:0, instance:com.mystore.testscripts.IndexPageTest@16eb3ea3]" started-at="2022-02-01T11:55:27 IST" name="verifyTitle" finished-at="2022-02-01T11:55:27 IST" duration-ms="279" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [null] but found [automationpractice.com]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [null] but found [automationpractice.com]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.mystore.testscripts.IndexPageTest.verifyTitle(IndexPageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTitle -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.mystore.testscripts.IndexPageTest@16eb3ea3]" started-at="2022-02-01T11:55:28 IST" name="tearDown" finished-at="2022-02-01T11:55:30 IST" duration-ms="1992" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:com.mystore.testscripts.IndexPageTest@16eb3ea3]" started-at="2022-02-01T11:55:30 IST" name="afterSuite" finished-at="2022-02-01T11:55:31 IST" duration-ms="641" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- com.mystore.testscripts.IndexPageTest -->
    </test> <!-- IndexPage -->
  </suite> <!-- Suite -->
</testng-results>
